CHANGELOG-developer.asciidoc

```diff
@@ -31,6 +31,7 @@ The list below covers the major changes between 6.3.0 and master only.
   used because the value is specified in magefile.go. {pull}7670[7670]
 - Outputs must implement String. {pull}6404[6404]
 - Renamed `-beat-name` CLI option used in `kibana_index_pattern.go` to `-beat` for consistency with other scripts in `dev-tools/cmd`. {pull}8615[8615]
+- Systemd unit file template used on Linux packaging includes now environment variables to ease flag overriding. One of them includes the `-e` flag, making beats log to stderr by default on systemd uses. {pull}8942[8942]
```

Comment:
"includes now" - Was the meant to be "includes new"? If not then I'd change it to "now includes".

Reply:
I meant "now includes" yes, thanks, I'll change it.